great_expectations/rule_based_profiler/parameter_builder/numeric_metric_range_multi_batch_parameter_builder.py

```diff
@@ -665,9 +691,68 @@ def _get_bootstrap_estimate(
         return compute_bootstrap_quantiles_point_estimate(
             metric_values=metric_values,
             false_positive_rate=false_positive_rate,
+            quantile_statistic_interpolation_method=quantile_statistic_interpolation_method,
             n_resamples=n_resamples,
             random_seed=random_seed,
+        )
+
+    @staticmethod
+    def _get_kde_estimate(
+        metric_values: np.ndarray,
+        domain: Domain,
+        variables: Optional[ParameterContainer] = None,
+        parameters: Optional[Dict[str, ParameterContainer]] = None,
+        **kwargs,
+    ) -> NumericRangeEstimationResult:
+        # Obtain n_resamples override from "rule state" (i.e., variables and parameters); from instance variable otherwise.
+        n_resamples: Optional[int] = get_parameter_value_and_validate_return_type(
+            domain=domain,
+            parameter_reference=kwargs.get("n_resamples"),
+            expected_return_type=None,
+            variables=variables,
+            parameters=parameters,
+        )
+
+        if n_resamples is None:
+            n_resamples = (
+                NumericMetricRangeMultiBatchParameterBuilder.DEFAULT_KDE_NUM_RESAMPLES
+            )
+
+        # Obtain bw_method override from "rule state" (i.e., variables and parameters); from instance variable otherwise.
+        bw_method: Optional[Any] = get_parameter_value_and_validate_return_type(
```

Comment:
@NathanFarmer Is the return type of `Any` the best we can expect here?  The property returns something else -- perhaps a typo?  Thanks.

Reply:
@alexsherstinsky This parameter accepts quite a few types. I will update this to `Optional[Union[str, float, Callable]]`

Reply:
I linked to the acceptable parameter values in the `NumericMetricRangeMultiBatchParameterBuilder` docstring (https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.gaussian_kde.html) and I updated this type hint everywhere. ✅